api/app/Policies/TeamPolicy.php

```diff
@@ -68,4 +68,19 @@ public function update(User $user = null)
         }
         return false;
     }
+
+    /**
+     * Determine whether the user can view the members of a team.
+     * Likely to be updated later to allow the team admin and teammates to view their own team
+     *
+     * @param  \App\Models\User|null  $user
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewAnyTeamMembers(User $user = null)
+    {
+        if ($user) {
+            return $user->isAdmin();
+        }
+        return false;
+    }
```

Comment:
Unless the policy needs to return true for guest users, you can leave off the `= null` in the argument, and assume its not null.
```suggestion
    public function viewAnyTeamMembers(User $user)
    {           
        return $user->isAdmin();
    }
```

Reply:
:+1:  [Make user model not-null](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5749/commits/31fcab5dbbbd25710506262e70b2f6483ad95de5)